ONSdigital/rm-comms-template-service
# Changed so can pass arguments for different environments

# Motivation and Context
Need to configure for different environments so changed to an argument based script

# What has changed
Passing in arguments now i.e. url, username and password

# How to test?
- Simply run `pipenv run python ./scripts/config_comms_templates.py http://localhost:8182 admin secret` to get templates into your db first.
- Then run`pipenv run python ./scripts/config_existing_comms_template.py http://localhost:8182 admin secret` to add the updated templates
- Check all templates are in your db based on JSON schemas that get loaded in.

# Links
Card: https://trello.com/c/pUPbT4Q0/336-ensure-all-comms-templates-are-configured-in-production-m
